app/scripts/models/account.js

```diff
@@ -870,17 +870,19 @@ const Account = Backbone.Model.extend({
      * @param {String} token - email verification token
      * @param {String} code - email verification code
      * @param {Object} relier - relier being signed in to.
+     * @param {String} emailToHashWith - use this email to hash password with.
      * @returns {Promise} - resolves when complete
      */
-  completePasswordReset (password, token, code, relier) {
+  completePasswordReset (password, token, code, relier, emailToHashWith) {
```

Comment:
Not a huge fan of this variable name, but I used it in other parts of the code..so I'll be consistent for time being.

Reply:
+1 for consistency, at least until we get around to a cleanup like https://github.com/mozilla/fxa-auth-server/issues/2344